networking/gcp/psa/variables.tf

```
@@ -0,0 +1,69 @@
+######## SECTION - VPC to get
+
+variable "vpc_network" {
+  description = "The vpc_network on which the psa will be created"
+  type        = string
+}
+
+######## SECTION - PSA 
+
+variable "service_name" {
+  description = "The private service access to create"
+  type        = string
+}
```

As most of the time on GCP, the service will be `servicenetworking.googleapis.com`, It would be better to have it as the default value:
```suggestion
variable "service_name" {
  description = "The private service access to create"
  type        = string
  default     = "servicenetworking.googleapis.com"
}
```